### ISSUE TITLE
Named Tuples raising warning S1854

### ISSUE THREAD

**Issue by HappyDump** [SonarSource/sonar-dotnet#3116]
**Description**
I'm getting a false positive on S1854

**Repro steps**

```
public (int, int, int) FooBar()
        {
            var foo = 2;

            return (foo * 2, foo, foo / 2);
        }
```

**Expected behavior**
No warning

**Actual behavior**

<img width="218" alt="FooBar" src="https://user-images.githubusercontent.com/3336054/74353109-bd64a680-4db9-11ea-8d57-9e1dd86bb833.png">

**Known workarounds**
n/a

**Related information**
SonarLint for VS Version: 4.17.0.14702
Visual Studio 2019 / C#

**Comment by andrei-epure-sonarsource** [comment]
thanks @HappyDump !
This is a duplicate of  #3094 , the fix will get in the next release

**Comment by HappyDump** [comment]
Thanks @andrei-epure-sonarsource  sorry for the duplicate, I didn't see that issue.
